pyproject.toml

```diff
@@ -92,6 +92,7 @@ gcp = [
     "google-cloud-build==3.24.1",
     "ml_goodput_measurement==0.0.2",
     "pyOpenSSL>=22.1.0",  # compat with cryptography version.
+    "pathwaysutils@git+https://github.com/google/pathways-utils", # for JAX+Pathways single-controller accelerator coordinator
```

Comment:
Let's put pathways-utils in a separate section, since it's not a critical dependency for regular GCP usage.

Reply:
This makes sense. Moving pathways to its own section.